e2e/testsuite/diagnostics/diagnostics.go

```diff
@@ -19,7 +19,7 @@ import (
 
 const (
 	e2eDir          = "e2e"
-	defaultFilePerm = 0750
+	defaultFilePerm = 0o750
```

Comment:
were there linting issues here? Just wondering as I didn't run into any issues when setting this variable 🤔

Reply:
These changed when I run gofumpt....